apps/rush-lib/src/api/VersionPolicy.ts

```diff
@@ -72,6 +79,16 @@ export abstract class VersionPolicy {
   constructor(versionPolicyJson: IVersionPolicyJson) {
     this._policyName = versionPolicyJson.policyName;
     this._definitionName = VersionPolicyDefinitionName[versionPolicyJson.definitionName];
+
+    const {
+      dependencies: {
+        versionFormatForCommit = VersionFormatForCommit.original,
+        versionFormatForPublish = VersionFormatForPublish.original
+      } = {}
+    } = versionPolicyJson;
```

Comment:
@iclanton  I just spent 10 minutes trying to figure out this obfuscated code.  :-P  When reviewing PRs we need to keep an eye out for this stuff, unless there's a tslint rule that can catch it.